ui/lib/sync/addon/components/secrets/page/overview.ts

```diff
@@ -15,25 +15,23 @@ import Ember from 'ember';
 import type FlashMessageService from 'vault/services/flash-messages';
 import type StoreService from 'vault/services/store';
 import type RouterService from '@ember/routing/router-service';
-import type VersionService from 'vault/services/version';
 import type { SyncDestinationAssociationMetrics } from 'vault/vault/adapters/sync/association';
 import type SyncDestinationModel from 'vault/vault/models/sync/destination';
 
 interface Args {
   destinations: Array<SyncDestinationModel>;
   totalVaultSecrets: number;
-  activatedFeatures: Array<string>;
+  secretsSyncPersona: string;
```

Comment:
plus one to that! 👆 since the personas are essentially magic strings that we're defining ourselves, it'd be helpful to use an enum as guardrails against passing bad input. 🙂

Reply:
Great idea!